test/electron-renderer/ui/modals/GoToModal_spec.js

```diff
@@ -74,6 +77,105 @@ createModalTest(GoToModal, ['gotourl'], [], [], () => {}, (_c) => {
     fired.generateDebugInfo.should.be.ok;
   });
 
+  describe('stubbed fetch request', () => {
```

Comment:
This is "context" to the nested `it` test.  Concatenate the strings to get what the logs will output.  This would make more sense as

```js
describe('with a redirecting URL', () => {`
```

Then in the logs you get

`GoToModal with a redirecting URL given URL should emit the ....`